### PR TITLE
[usecase] Add WebCL usecase

### PR DESCRIPTION
**Pull request by JianfengXu**
- the WebCL is not supported now, so labeled as designed

### CODE REVIEW

File: usecase/usecase-webapi-xwalk-tests/samples/WebCL/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2013 Intel Corporation.
```
Comment (zqzhang) [inline usecase/usecase-webapi-xwalk-tests/samples/WebCL/index.html]: s/2013/2015/g

File: usecase/usecase-webapi-xwalk-tests/samples/WebCL/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+<script src="../../js/bootstrap.min.js"></script>
+<script src="../../js/common.js"></script>
+<script src="../../js/tests.js"></script>
+
+<div id="header">
+    <h3 id="main_page_title"></h3>
+</div>
+<div class="content">
+  <h4>This sample demonstrates the basic WebCL operation.</h4>
+  <div id="d" style="height:200px">
```
Comment (zqzhang) [inline usecase/usecase-webapi-xwalk-tests/samples/WebCL/index.html]: I personally hate any inline `style`s

File: usecase/usecase-webapi-xwalk-tests/samples/WebCL/index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Xu, Jianfeng <jianfengx.xu@intel.com>
+
+-->
+
+<meta charset="utf-8" />
+<meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, width=device-width">
+<link rel="stylesheet" type="text/css" href="../../css/bootstrap.css">
```
Comment (zqzhang) [inline usecase/usecase-webapi-xwalk-tests/samples/WebCL/index.html]: No need ` type="text/css"`

File: usecase/usecase-webapi-xwalk-tests/samples/WebCL/res/COPYING [hunk 4]
```diff
@@ -0,0 +1,5 @@
+index.html come from
+https://github.com/uebquite/webcl-demos
```
Comment (zqzhang) [inline usecase/usecase-webapi-xwalk-tests/samples/WebCL/res/COPYING]: Actually this is https://github.com/uebquite/webcl-demos/blob/master/Hello/index.html

File: usecase/usecase-webapi-xwalk-tests/samples/WebCL/res/COPYING [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+
+These tests are copyright by the Khronos Group under MIT License:
+https://www.khronos.org/registry/webgl/sdk/tests/test-guidelines.md
```
Comment (zqzhang) [inline usecase/usecase-webapi-xwalk-tests/samples/WebCL/res/COPYING]: However, I cannot agree to this license for https://github.com/uebquite/webcl-demos/blob/master/Hello/index.html

They are totally 2 different projects! And I find no license from https://github.com/uebquite/webcl-demos

**Comment by zqzhang** [comment]
God bless me. Cannot merge this now.

@JianfengXu could you please update it and ping @jennycao to review and merge it?

**Comment by zqzhang** [comment]
merge as-is.

**Comment by jennycao** [comment]
@zqzhang @JianfengXu This PR should not be merged since the liense issue.
we found another sample with liense https://github.com/uebquite/webcl-demos/tree/master/Deform 

So I will convert this PR, and jianfeng will replace it with one.

**Pull request merged**